client/src/components/AddressCard.jsx

```diff
@@ -6,284 +6,302 @@ import Typography from "@mui/material/Typography";
+    baths,
+    price,
+    late_payments,
+    listing_link,
+  } = address || {};
 
-    setReviewInfo({
-      overall: "4.5",
-      respectfulness: "2",
-      responsiveness: "5",
-      tlc: "4.5",
-      personalization: "3",
-    });
-
-    setCriteria({
-      dogSizeMax: "20lbs",
-      breedRestriction: true,
-      dogsAllowed: true,
-      petsAllowed: true,
-      bikesAllowed: false,
-      overnightGuests: true,
-      quietHours: "10pm-8am",
-      smoking: false,
-      latePayment: "fees included",
-    });
-  }, []);
+  const postAddress = async () => {
+    await fetch(`/address/`, {
+      method: "POST",
+      body: JSON.stringify(addressForm),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => {
+        window.location.reload(false);
+      })
+      .catch((err) => {
+        console.log("error submitting address ->" + err);
+      });
+  };
 
   return (
-    <Card
-      sx={{maxWidth: 390, backgroundColor: "#002147", color: "#ededed" }}
-      id="addressCard"
-    >
-      {/* title container */}
-      <div
-        style={{
-          display: "flex",
-          flexDirection: "column",
-          justifyContent: "flex-end",
-          alignItems: "center",
-          padding: "15px 15px 5px 15px",
-          letterSpacing: "2px",
-        }}
-      >
-        <div style={{ fontSize: "20px" }}>
-          {`${homeInfo.streetNum} ${homeInfo.street} #${homeInfo.apartmentNum}`}
-        </div>
-        <Typography
-          sx={{ fontSize: "10px", letterSpacing: "5px", color: "#bababa" }}
-        >
-          {homeInfo.buildingType}
-        </Typography>
-      </div>
-
-      {/* picture */}
-      <div id="addressPic">
-        <CardMedia
-          image="/images/address_1.jpg"
-          alt="home"
-          height="200"
-          component="img"
-          sx={{ zIndex: 0 }}
-        />
-
-        {/* price box */}
-        <Box
-          id="price"
-          sx={{
-            position: "absolute",
-            flexDirection: "column",
-            zIndex: 1,
-            backgroundColor: "#002147",
-            opacity: 0.6,
-            dropShadow: "20px 20px 20px black",
-            borderRadius: "0px 0px 5px 0px",
-            padding: "5px",
-          }}
-        >
-          ${homeInfo.price}
-        </Box>
-
-        {/* beds and baths box */}
-        <Box
-          id="bedsBaths"
-          sx={{
-            position: "absolute",
-            display: "flex",
-            flexDirection: "column",
-            justifyContent: "center",
-            alignItems: "center",
-            zIndex: 1,
-            right: 0,
-            top: 0,
-            backgroundColor: "#002147",
-            opacity: 0.6,
-            borderRadius: "0px 0px 0px 5px",
-            padding: "0px 0px 5px 5px",
-            width: "70px",
-          }}
-        >
-          <div
-            style={{
+    <>
+      {isAddCard &&
+        (addPressed ? (
```

Comment:
need more clarification on addPressed. Is this a future conditional render?

Reply:
yes, for when a landlord clicks the plus icon on their profile to add an address